dome9CloudBots/bots/storage_account_enable_https_traffic_only.py

```diff
@@ -0,0 +1,34 @@
+# What it does: enable Storage account Secure transfer required field
+# Usage: AUTO: storage_account_enable_https_traffic_only
+# Limitations: None
+
+import logging
+from msrestazure.azure_exceptions import CloudError
+from azure.mgmt.storage import StorageManagementClient
+from azure.mgmt.storage.models import StorageAccountUpdateParameters
+
+
+def run_action(credentials, rule, entity, params):
```

Comment:
1. Why `params` and `rule` are not in use?
2. You are using Python 3, so FYI it would be much more readable if you would use annotations

```
from typing import Dict
def run_action(... , entity: Dict, ...):
```
Is much more readable.